README.md

```diff
@@ -102,6 +102,16 @@ omnect-cli iot-hub-device-update import-update --help
 
 **Note**: The import process may take several minutes.
 
+### Remove update from IoT Hub
+This command removes an update from Azure Device Update for IoT Hub that was previously imported by `import-update` command.
```

Comment:
just a notice: omnect-cli could remove updates that are created manually...
->  delete? "that was previously imported by `import-update` command"

Reply:
I just want to document workflows in context of omnect-cli. In this case the reader should just know that he has to import an update before he can remove one.